calour/transforming.py

```diff
@@ -362,6 +362,12 @@ def subsample_count(exp: Experiment, total, replace=False, inplace=False):
        should be discrete count. The samples that have few total count
        than ``total`` will be dropped.
 
+    .. note:: This function may not work on Windows OS. It relies on
+       the :func:`skbio.stats.subsample_counts` which have
+       `ValueError: Buffer dtype mismatch, expected 'int64_t' but got
+       'long'` in `_subsample_counts_without_replacement` function of
+       `skbio/stats/__subsample.pyx`
```

Comment:
do we have an issue about this in skbio? if so, maybe worth adding the issue number here? if not - worth adding an issue there?